.github/workflows/docker-build.yaml

```diff
@@ -3,13 +3,23 @@ name: build & deploy
 on:
   push:
     branches:
-      - "develop"
+      - "*"
```

Comment:
doesn't look right to me.

Build should only be triggered by certain events, not all pushes.